src/util/fetchUtils.js

```diff
@@ -87,8 +87,8 @@ export const getFetchArgs = (apiUrl, queries, meta) => {
 	const isFormData = queries[0].params instanceof FormData;
 	const isDelete = method === 'DELETE';
 
-	const fetchUrl = new URL(apiUrl);
-	fetchUrl.searchParams.append('queries', rison.encode_array(queries));
+	const searchParams = new URLSearchParams();
```

Comment:
Does this part only run in the browser, or does `window.location` get populated on the server?

Curious if it's the latter, because this might not work. We're routing traffic on GAE a little differently, which I think is what was causing `info.host` to be the wrong thing (we are setting a host header on requests to new platform requests on www.meetup.com to the appspot url).